src/main/java/TestsForLecture10/FileUploadTest.java

```diff
@@ -0,0 +1,28 @@
+package TestsForLecture10;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.time.Instant;
+
+public class FileUploadTest extends BaseTest {
+
+
+
+    @BeforeMethod
+    public void navigate() {
+        driver.get("http://the-internet.herokuapp.com/upload");
+    }
+    @Test
+    public void uploadFileTest(){
+        WebElement selectFileButton = driver.findElement(By.cssSelector("#file-upload"));
+        selectFileButton.sendKeys(System.getProperty("D:/study/JAVA/automation_testing_java_course/firstProject/HerokuApp_Sergey_Glinisty/src/main/resources")+"src/main/resources/animals-world-banner.jpg");
```

Comment:
Please use relevant path instead of absolute:
System.getProperty("user.dir") + "/src/test/resources/path_to_file..."

Reply:
Didn't understand, instead "D:/study/JAVA/automation_testing_java_course/firstProject/HerokuApp_Sergey_Glinisty/src/main/resources" should be "user.dir"? and 
How does user.dir find out the path to the file on my laptop?

Reply:
System.getProperty("user.dir") returns the path from disk root to Project folder on any PC, then you add relative path to file starting from project root "src/...."